fsevents.js

```diff
@@ -6,7 +6,11 @@
 /* jshint node:true */
 'use strict';
 
-var Native = require('./build/Release/fse');
+var binary = require('node-pre-gyp');
+var path = require('path');
+var binding_path = binary.find(path.resolve(path.join(__dirname,'./package.json')));
+var Native = require(binding_path);
```

Comment:
https://github.com/mapbox/node-pre-gyp/blob/master/lib/pre-binding.js#L15-L25

It does pretty much just get passed straight to `require`. I don't see why simply `binary.find('package.json')` wouldn't work.